code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -138,14 +139,47 @@
 
 	wound_overlay.vis_flags &= ~VIS_HIDE // Show the overlay
 
+/mob/living/carbon/xenomorph/proc/update_snowflake_overlays()
+	if(!backpack_overlay)
+		return
+	if(!back)
```

Comment:
This is redundant, just have the istype check, it'll catch it being null as well